### PR TITLE
Adapt therapeutics abstract for new format

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This is a draft updated abstract. The "Abstract" part is about 30 words too long, so definitely open to suggestions!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -2,8 +2,25 @@
 
 ### Abstract
 
+After emerging in China in late 2019, the novel _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021, continues to significantly impact most countries.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:5]:
```suggestion
After emerging in China in late 2019, the novel _severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021, continues to significantly impact most countries.
```

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -2,8 +2,25 @@
...
+Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:6]:
```suggestion
Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _severe acute respiratory syndrome coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome coronavirus_, which emerged in 2012.
```

Comment (RLordan) [reply content/20.pharmaceuticals.md:6]: You can correct me if I'm wrong but no need for the related part I think

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -2,8 +2,25 @@
...
+Both of these previous epidemics were controlled fairly rapidly through public health measures.
+However, previous insights into the immune response to coronaviruses gained during the outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have proved beneficial to identifying approaches to the treatment and prophylaxis of novel coronavirus disease 2019 (COVID-19).
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:8]:
```suggestion
However, previous insights into the immune response to coronaviruses gained during the outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have proved beneficial for identifying approaches to treat and prevent novel coronavirus disease 2019 (COVID-19).
```

**Review by RLordan — APPROVED** [review]
Thanks @rando2

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:6]: I'm sorry I'm always shooting you down on this but I think this is how they want it, based on their Excel sheet of virus names! I'm so sorry that I never incorporate these changes, we'll see if my interpretation of the style guide is correct 😂

Comment (RLordan) [reply content/20.pharmaceuticals.md:6]: Haha this is funny! No problem, you know I will totally forget and say it again haha

**Pull request merged**